crates/air-lib/polyplets/src/lib.rs

```diff
@@ -25,6 +25,16 @@
     unreachable_patterns
 )]
 
+extern crate self as marine_rs_sdk;
```

Comment:
Wow, why is that needed?

Reply:
#[marine] generates code that uses marine_rs_sdk::internal, so it is needed to provide it without depending on marine_rs_sdk

Reply:
I see. Clever trick! I think it deserves a comment.